frontend/src/utils/format.ts

```diff
@@ -108,3 +109,26 @@ export function formatShipmentName(
   const month = shipment.labelMonth.toString().padStart(2, '0')
   return `${shipment.shippingRoute}-${shipment.labelYear}-${month}`
 }
+
+export function formatContainerType(type: LineItemContainerType) {
+  return {
+    [LineItemContainerType.Box]: 'Box',
+    [LineItemContainerType.BulkBag]: 'Bulk bag',
+    [LineItemContainerType.FullPallet]: 'Full pallet',
+    [LineItemContainerType.Unset]: 'Not set',
+  }[type]
+}
+
+export function getLineItemVolume(lineItem: any) {
```

Comment:
suggestion (non-blocking): rename `getLineItemVolume` with the unit suffix `getLineItemVolumeInSquareMeters`